### PR TITLE
Populate final pages for chapter 10

### PR DESCRIPTION
**Pull request by benalleng**
making a payment 6 and the end screen were not correctly implemented.

### CODE REVIEW

File: utils/data.ts [hunk 1]
```diff
@@ -136,11 +136,11 @@ export const transactionTabs: TransactionData = {
     },
     output_0: {
       sats: '2000',
-      script: `T1BfSUbigKhPUF9QVVNIIDcwMCBPUF9DSEVDS1NFUVVFTkNFVkVSSUZZIE9QX0RST1DigKhPUF9QVVNIIFBVQktFWShMYXN6bG8pIE9QX0NIRUNLU0lH4oCoT1BfRUxTReKAqE9QXzIgT1BfUFVTSCBQVUJLRVkocmV2b2NhdGlvbl9sYXN6bG9fMSkgT1BfUFVTSCBQVUJLRVkoWW91KSBPUF8yIE9QX0NIRUNLTVVMVElTSUfigKhPUF9FTkRJRg==`,
```
Comment (benalleng) [inline utils/data.ts:139]: I think I messed this up

**Comment by satsie** [comment]
Hah, we were working on that last screen at the same time! :laughing: https://github.com/saving-satoshi/saving-satoshi/pull/1140

I'm going to close my PR in favor of yours, but I still have a question about the "End" button. We're supposed to have a button to The Bitcoin Dev Project. Should we just update the End button to go there? Change the text to something like "Begin your next adventure" (ideally something better than this phrasing)? cc @GBKS 

Screenshot from 2024-10-21 16-43-59

**Comment by benalleng** [comment]
I just replaced `End` with a duplicate link. I figure if they want to go back to the chapters page they can just press the power button at that point.

**Review by satsie — APPROVED** [review]
I can't comment on the correctness of the challenge answers but everything else looks good to me!

**Pull request merged**